crates/ruma-events/src/enums.rs

```diff
@@ -414,7 +421,34 @@ impl AnyMessageLikeEventContent {
                 Some(encrypted::Relation::Reference(relates_to.clone()))
             }
             #[cfg(feature = "unstable-msc3381")]
-            Self::PollStart(_) | Self::UnstablePollStart(_) => None,
+            Self::UnstablePollStart(UnstablePollStartEventContent::New(content)) => {
+                match &content.relates_to {
+                    Some(RelationWithoutReplacement::Thread(thread)) => {
+                        Some(encrypted::Relation::Thread(thread.clone()))
+                    }
+                    Some(RelationWithoutReplacement::Reply { in_reply_to }) => {
+                        Some(encrypted::Relation::Reply { in_reply_to: in_reply_to.clone() })
+                    }
+                    Some(RelationWithoutReplacement::_Custom(_)) | None => None,
+                }
+            }
```

Comment:
For this is would probably be nicer to implement `From<RelationWithoutReplacement> for Relation` (`Relation` being the one in `room::encrypted`), then we can do here:

```suggestion
            Self::UnstablePollStart(UnstablePollStartEventContent::New(content)) => {
                content.relates_to.clone().map(Into::into)
            }
```

Reply:
Where should I add this? To the same file where `RelationWithoutReplacement` is declared or in this one?

Reply:
To the file where `room::encrypted::Relation` is declared I'd say.